### DISCUSSION TITLE
Question about this

### DISCUSSION THREAD

**Discussion by ziomek64** [AniTrend/support-markdown#248] [Q&A]
Is this already working or far from it? If so, what is working from anilist tags?

**Comment by wax911** [comment]
> Is this already working or far from it? If so, what is working from anilist tags?

Not entirely sure how to answer this to be honest, it works for the use-cases it covers. Have you tried running the project?

**Reply by ziomek64** [reply]
Well yeah, crashes for me

**Reply by wax911** [reply]
Strange, I wasn't aware that it stopped working. I'll see if I can have a look at it this week

**Reply by wax911** [reply]
@ziomek64 fixed in https://github.com/AniTrend/support-markdown/pull/251
